doc/nrf/working_with_nrf/nrf53/nrf5340_gs.rst

```diff
@@ -0,0 +1,109 @@
+.. _ug_nrf5340_gs:
+
+Getting started with nRF5340 DK
+###############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This section gets you started with your nRF5340 :term:`Development Kit (DK)` using the |NCS|.
+It tells you how to install the :ref:`peripheral_uart` sample and perform a quick test of your DK.
+
+If you have already set up your nRF5340 DK and want to learn more, see the following documentation:
+
+* :ref:`ug_nrf5340` for more advanced topics related to the nRF5340 DK if you are already familiar with the |NCS|.
+* The :ref:`introductory documentation <getting_started>` for more information on the |NCS| and the development environment.
+
+.. _nrf5340_gs_requirements:
+
+Minimum requirements
+********************
+
+Make sure you have all the required hardware and that your computer has one of the supported operating systems.
+
+Hardware
+========
+
+* nRF5340 DK
+* micro-USB 2.0 cable
+
+Software
+========
+
+On your computer, one of the following operating systems:
+
+* Microsoft Windows 10
+* macOS X, latest version
+* Ubuntu Linux, latest Long Term Support (LTS) version
+
+On your mobile device, one of the following operating systems:
+
+* Android
+* iOS
+
+.. _nrf5340_gs_installing_software:
+
+Installing the required software
+********************************
+
+On your computer, install `nRF Connect for Desktop`_.
+After installing and starting the application, install the Programmer app.
+
+You must also install a terminal emulator, such as :ref:`PuTTY <putty>` or the nRF Terminal, which is part of the `nRF Connect for Visual Studio Code`_ extension.
+The steps detailed in :ref:`nrf5340_gs_connecting` use PuTTY, but any terminal emulator will work.
```

Comment:
Do we want to highlight PuTTY steps instead of nRF Terminal? I thought we were strongly encouraging users to use the VSC extension.

Reply:
That could be good. The original version was using Termite, and I updated to PuTTY since that had documentation in the NCS but Termite didn't. But you are right, nRF Terminal is probably what we should be encouraging.

Reply:
I'll create a separate PR for that change since it'll be in the NRF52 Series file, to not muddy this PR too much.